pkg/tnf/handlers/handler_template/handler.tmpl

```diff
@@ -55,28 +65,28 @@ func (h *{{ .UpperHandlername }}) Result() int {
 	return h.result
 }
 
-// ReelFirst returns a step which expects an {{ .UpperHandlername }} summary for the given device.
+// ReelFirst returns a reel step for handler {{ .UpperHandlername }}.
 func (h *{{ .UpperHandlername }}) ReelFirst() *reel.Step {
 	return &reel.Step{
-		Expect: []string{""}, // TODO : pass the list of possible regex in here
+		Expect:  []string{}, // TODO : pass the list of possible regex in here
 		Timeout: h.timeout,
 	}
 }
 
 // ReelMatch parses the {{ .UpperHandlername }} output and set the test result on match.
 func (h *{{ .UpperHandlername }}) ReelMatch(_, _, match string) *reel.Step {
+	// TODO : add the matching logic here and return an appropriate tnf result.
 	h.result = tnf.ERROR
 	return nil
-	// TODO : add the matching logic in here
 }
 
-// ReelTimeout does nothing, {{ .UpperHandlername }} requires no explicit intervention for a timeout.
+// ReelTimeout function for {{ .UpperHandlername }} will be called by the reel FSM when a expect timeout occurs.
 func (h *{{ .UpperHandlername }}) ReelTimeout() *reel.Step {
+	// TODO : Add code here in case a timeout reaction is needed.
 	return nil
-	// TODO : fill the stub
 }
 
```

Comment:
general question, does the code compile properly after generation ?

Reply:
Yes, somehow it worked.